label_sync/labels.md

```diff
@@ -47,6 +47,7 @@ larger set of contributors to apply/remove them.
 
 | Name | Description | Added By | Prow Plugin |
 | ---- | ----------- | -------- | --- |
+| <a id="api-review" href="#api-review">`api-review`</a> | Categorizes an issue or PR as actively needing an API review. See https://git.k8s.io/community/blob/master/sig-architecture/api-review-process.md| anyone |  [label](https://git.k8s.io/test-infra/prow/plugins/label) |
```

Comment:
This needs to be https://git.k8s.io/community/sig-architecture/api-review-process.md :)

Reply:
oops, updated